ansible_base/jwt_consumer/common/auth.py

```diff
@@ -143,9 +146,13 @@ def parse_jwt_token(self, request):
 
         logger.info(f"User {self.user.username} authenticated from JWT auth")
 
-    def log_and_raise(self, conditional_translate_object, expand_values={}):
+    def log_and_raise(self, conditional_translate_object, expand_values={}, error_code=None):
         logger.error(conditional_translate_object.not_translated() % expand_values)
-        raise AuthenticationFailed(conditional_translate_object.translated() % expand_values)
+        translated_error_message = conditional_translate_object.translated() % expand_values
+        if error_code == 498:
```

Comment:
498 is used in a couple places, do you want to make a constant incase we ever want to change this number?